wordpress-mobile/WordPress-iOS
# Jetpack Section: Add Warning and Restore Status screens

Part of #15191 

- Adds Warning skeleton screen
- Adds Restore Status skeleton screen

⚠️ **The following will be addressed in subsequent PRs:**
- Passing data
- API hookup
- Styling

### To test:

1. My Site > Activity Log > rewindable activity cell
2. Tap on ellipsis icon
3. Tap on `Restore`
4. Tap on `Restore to this point`
     - ✅ Should display new Warning screen
 5. Tap on `Confirm`
     - ✅ Should display new Restore Status screen

Warning | Restore Status
--- | ---
![Screen Shot 2021-01-07 at 20 48 57](https://user-images.githubusercontent.com/6711616/103889892-7d718500-512a-11eb-81c8-17eb8cc572da.png) | ![Screen Shot 2021-01-07 at 20 49 00](https://user-images.githubusercontent.com/6711616/103889897-7fd3df00-512a-11eb-9ff9-a15e80aee1e1.png)


### PR submission checklist:
